docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -297,6 +297,16 @@ The total number of snapshots that match the request when ignoring size limit or
 The number of remaining snapshots that were not returned due to size limits and that can be fetched by additional requests using the `next`
 field value.
 
+`search`::
+(string)
+A search string to filter snapshots by. The search string can specify a query against the fields `repository` (repository name),
+`name` (snapshot name) or `policy` (the SLM policy name that created the snapshot if it was created by SLM).
+Supported operations are `=` for exact text match and `:` for partial text match. Starting the query string on a `-`, means that all
```

Comment:
It'd help a great deal to have a bunch of examples here.

Reply:
++ will add some more